assets/models/system/deepset-minilm-uncased-squad2/description.md

```diff
@@ -31,7 +31,7 @@ Question Answering|Extractive Q&A|<a href="https://huggingface.co/datasets/squad
 #### Sample input
 ```json
 {
-    "inputs": {
+    "input_data": {
         "question": ["What is my name?", "Where do I live?"],
         "context": ["My name is John and I live in Seattle.", "My name is Ravi and I live in Hyderabad."]
     }
```

Comment:
This example seems confusing, could we use a better one?

Reply:
@skasturi @novaturient95  can you please comment is this wrong or do we need to change the input data